views/js/runner/plugins/controls/connectivity/connectivity.js

```diff
@@ -84,7 +84,10 @@ define([
                 }
 
                 //offline navigation error, we pause the test
-                if(err.source === 'navigator' && err.purpose === 'proxy' && err.code === 404){
+                if (
+                    proxy.isOffline() ||
+                    (err.source === 'navigator' && err.purpose === 'proxy' && err.code === 404)
```

Comment:
I don't really see how this could be improved, I'd leave it like that

Reply:
Ok, thanks for looking @dietertaotesting. I'm going to try to give it one last review today and I'll probably assign it to you tomorrow.